2007-07-24  Jakub Jelinek  <jakub@redhat.com>

	* scripttempl/elf.sc: Add .note.gnu.build-id.

--- ld/scripttempl/elf.sc.jj	2007-07-24 10:07:02.000000000 +0200
+++ ld/scripttempl/elf.sc	2007-07-24 21:44:35.000000000 +0200
@@ -267,6 +267,7 @@ SECTIONS
   ${CREATE_SHLIB+${RELOCATING+. = ${SHLIB_TEXT_START_ADDR:-0} + SIZEOF_HEADERS;}}
   ${CREATE_PIE+${RELOCATING+. = ${SHLIB_TEXT_START_ADDR:-0} + SIZEOF_HEADERS;}}
   ${INITIAL_READONLY_SECTIONS}
+  .note.gnu.build-id : { *(.note.gnu.build-id) }
   ${TEXT_DYNAMIC+${DYNAMIC}}
   .hash         ${RELOCATING-0} : { *(.hash) }
   .gnu.hash     ${RELOCATING-0} : { *(.gnu.hash) }
